conference/src/main/java/com/kyriba/conference/management/api/HallController.java

```diff
@@ -43,45 +44,45 @@ public HallController(HallService hallService)
 
   @ApiOperation(value = "Get a conference hall if exists")
   @GetMapping(value = "/{id}", produces = APPLICATION_JSON_UTF8_VALUE)
-  HallResponse findHall(@Valid @ApiParam(value = "Hall identity", required = true) @PathVariable Long id)
+  ResponseEntity<HallResponse> findHall(
```

Comment:
Why do you use ResponseEntity?

Reply:
I use it to return 404 if 'hallService.findHall(id)' is Optional.empty().
Removed.